resources/views/app/compatible-wallets.blade.php

```diff
@@ -0,0 +1,57 @@
+@component('layouts.app')
+    <x-metadata page="wallets" />
+
+    @section('content')
+        <x-ark-container>
+            <div>
+                <h1 class="text-2xl font-semibold text-theme-secondary-900">@lang('pages.compatible-wallets.title')</h1>
+                <span class="text-sm">@lang('pages.compatible-wallets.subtitle')</span>
+            </div>
+
+            <div class="flex flex-col mt-6 w-full rounded-xl border min-[960px]:flex-row border-theme-secondary-300 dark:border-theme-secondary-800">
+                <div class="flex flex-col flex-1 justify-between py-8 px-8 space-y-6">
+                    <span class="text-sm items-center rounded font-semibold px-2 py-1 flex space-x-2 text-theme-secondary-900 dark:text-white dark:from-theme-navy-600 dark:via-theme-navy-600 dark:to-theme-navy-600 bg-gradient-to-r from-[#E5F0F8] via-[#FFF8EB] to-[#E5F0F8]">
+                        <x-ark-icon name="circle.info" />
+                        <span>@lang('pages.compatible-wallets.arkvault.disclaimer')</span>
+                    </span>
+                    <div>
+                        <h2 class="text-2xl font-semibold text-theme-secondary-900">@lang('general.arkvault') <span class="text-theme-secondary-500 dark:text-theme-secondary-700">(@lang('pages.compatible-wallets.arkvault.web_wallet'))</span></h2>
+                        <p class="mt-2 leading-7 dark:text-theme-secondary-400">@lang('pages.compatible-wallets.arkvault.description')</p>
+                    </div>
+                    <div class="flex flex-col justify-between py-3 pr-3 pl-6 rounded-xl sm:flex-row bg-theme-primary-50 dark:bg-theme-dark-blue-800">
+                        <div class="flex flex-1 items-center py-1 bg-no-repeat bg-right sm:dark:bg-[url('/images/wallets/arrows-dark.svg')] sm:bg-[url('/images/wallets/arrows.svg')] min-[960px]:bg-none lg:dark:bg-[url('/images/wallets/arrows-dark.svg')] lg:bg-[url('/images/wallets/arrows.svg')] mr-2">
+                            <div><x-ark-icon name="app.wallets-arkvault" size="none" class="w-10 h-10 dark:text-white text-theme-navy-600" /></div>
+                            <div class="flex flex-col ml-3">
+                                <span class="text-lg font-semibold dark:text-white text-theme-secondary-900">@lang('general.arkvault')</span>
+                                <span class="text-sm font-semibold text-theme-secondary-700 dark:text-theme-dark-blue-400">@lang('pages.compatible-wallets.arkvault.subtitle')</span>
+                            </div>
+                        </div>
+                        <div class="flex items-center">
+                            <a href="@lang('pages.compatible-wallets.arkvault.url')" target="_blank" rel="noopener nofollow noreferrer" class="flex items-center mt-4 w-full h-full rounded-xl sm:mt-0 sm:w-auto md:mt-0 md:w-full lg:w-auto button-primary"><div class="flex justify-center items-center h-full"><span>@lang('actions.learn_more')</span></div></a>
```

Comment:
Hard to read :sadmici: if it needs no new lines to handle annoying auto-spacing you could try `inline` or `inline-flex`

```suggestion
                            <a 
                                href="@lang('pages.compatible-wallets.arkvault.url')"
                                target="_blank"
                                rel="noopener nofollow noreferrer"
                                class="flex items-center mt-4 w-full h-full rounded-xl sm:mt-0 sm:w-auto md:mt-0 md:w-full lg:w-auto button-primary"
                            >
                                <div class="flex justify-center items-center h-full">
                                    <span>@lang('actions.learn_more')</span>
                                </div>
                            </a>
```